demo/index.html

```diff
@@ -32,7 +32,7 @@ <h1>IPA-JFK</h1>
     <p>GitHub: <a href="https://github.com/b1f6c1c4/IPA-JFK">b1f6c1c4/IPA-JFK</a></p>
 
     <p id="loading">Loading...</p>
-    <form action="#" style="display: none" id="frm">
+    <form action="#" id="frm">
```

Comment:
Why?

Reply:
Currently, this doesn't seem to have any visual effect. Even worse, in the most recent CI failure, `parcel build` was failed by this because `parcel` didn't know how to parse the inline style attributes.
This may be added back later if the migration is successful.

Reply:
The purpose of `style="display: none"` is to hide the form before finish loading the dictionary. BTW could you please point me  the exact line of failure?

Reply:
https://app.netlify.com/sites/ipa-jfk/deploys/6147af1a1db2a20008f2ba2f#L268
Yet it was not enough to indicate the exact line of failure with the `index.html`.

Reply:
OK you are right `parcel` is too silly with this. You shall figure out a workaround (but not showing the form before loading dictionary) before merging this PR :joy:

Reply:
That's truly an idiot.🙃 But wait, I just figured out something a few minutes ago in order to proceed with both `parcel` v2 and your original `html`.